### PR TITLE
make control plane mTLS for vms by default

### PR DESCRIPTION
**Pull request by rkpagadala**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
VM's communication with Pilot and Mixer is encrypted via Mutual TLS
```

### CODE REVIEW

File: install/tools/setupMeshEx.sh [hunk 1]
```diff
@@ -86,10 +89,15 @@ function istioDnsmasq() {
 # - name of the k8s cluster.
 function istioClusterEnv() {
    local K8S_CLUSTER=${1:-${K8S_CLUSTER}}
+   local ISTIO_NS=${ISTIO_NAMESPACE:-istio-system}
+   local CP_AUTH_POLICY=${CONTROL_PLANE_AUTH_POLICY:-MUTUAL_TLS"
```
Comment (ldemailly) [inline install/tools/setupMeshEx.sh]: missing close }

File: install/tools/setupMeshEx.sh [hunk 2]
```diff
@@ -86,10 +89,15 @@ function istioDnsmasq() {
...
 
    # TODO: parse it all from $(kubectl config current-context)
    CIDR=$(gcloud container clusters describe ${K8S_CLUSTER} ${GCP_OPTS:-} --format "value(servicesIpv4Cidr)")
    echo "ISTIO_SERVICE_CIDR=$CIDR" > cluster.env
+   echo "ISTIO_SYSTEM_NAMESPACE=$ISTIO_NS" >> cluster.env
```
Comment (ldemailly) [inline install/tools/setupMeshEx.sh]: who/what is using those?

Comment (rkpagadala) [reply install/tools/setupMeshEx.sh]: This is needed to construct discovery address, it is extracted by pilot-agent and used to construct the uri to verify for mTLS

**Pull request merged**